docs/USER_GUIDE.md

```diff
@@ -45,20 +45,17 @@ Endpoints]:
 
 #### Creating a New VPC
 
-If you do not have an existing VPC you wish to use, a VPC template is available
-as part of this solution which can be deployed separately to the main stack.
-This template will create a VPC with private subnets and all the relevant VPC
-Endpoints required by the Amazon S3 Find and Forget solution. To deploy this
-template, use the VPC Template "Deploy to AWS button" in
-[Deploying the Solution](#deploying-the-solution) then follow steps 5-9. The
-**Outputs** tab will contain the subnets and security group IDs to use as inputs
-for the main stack.
+If you do not have an existing VPC you wish to use, a solution provided VPC can
+be created by setting _DeployVpc_ to true when deploying the solution. The
+solution provided VPC only has private subnets and has all the relevant VPC
+Endpoints required by the Amazon S3 Find and Forget solution.
 
 #### Using an Existing VPC
 
 If you wish to use an existing VPC in your account with the Amazon S3 Find and
```

Comment:
I suggest replacing this paragraph with the following to provide better guidance on why you would want to use your own VPC, and elaborate on the parameters that need to be provided:
```suggestion
Amazon S3 Find and Forget can also be used in an existing VPC. You may
want to do this if you have requirements that aren't met by using the VPC
provided with the solution.

To use an existing VPC, set the `DeployVpc` parameter to `false` when
launching the solution CloudFormation stack. You must also specify the
subnet and security groups that the Fargate tasks will use by setting the
`VpcSubnets` and `VpcSecurityGroups` parameters respectively.

The subnets and security groups that you specify must allow the tasks to
connect to the aforementioned AWS services.
```